src/core/ColorPicker/ColorInputPanel.tsx

```diff
@@ -63,11 +64,27 @@ export const ColorInputPanel = React.forwardRef(
     React.useEffect(() => {
       if (currentFormat === 'hsl') {
         const hsl = activeColor.toHslColor();
+        const a = hsl.a
+          ? hsl.a.toFixed(2)
+          : (activeColor.getAlpha() / 255).toFixed(2);
```

Comment:
You could simplify this line by removing the ternary and using `??`.

```ts
const a = hsl.a?.toFixed(2) ?? (activeColor.getAlpha() / 255).toFixed(2);
```

Also this 255 part is really weird and we might need to refactor ColorValue soon, because alpha should only be between 0-1 or 0-100%

Reply:
Done